src/test/tests/rendering/offscreensave.py

```diff
@@ -123,4 +129,35 @@
 swa.height=300
 Test("offscreen_11",swa)
 
+# Test saving a 16384 x 16384 image
+slider.Delete()
+DeleteAllPlots()
+
+OpenDatabase(silo_data_path("curv2d.silo"))
+
+AddPlot("Pseudocolor", "d")
+DrawPlots()
+
+view2=GetView2D()
+view2.fullFrameActivationMode=view2.Off
+SetView2D(view2)
+
+swa.width = 16384
+swa.height = 16384
+swa.fileName = "image_16384x16384"
+swa.family = 0
+SetSaveWindowAttributes(swa)
+try:
+    SaveWindow()
+except:
+    TestValueEQ("md5 hash for 16384x16384 image", "Save failed", "9196b516c25ecbeac1fab4cd54ee0c59")
+
+# Comparing md5 sum instead of image, since the image is large.
```

Comment:
Cool way to not store a 600 Mb image to our tests 😄 . Am wondering a few things about testing a large image...

* wondering if there is any reason to test other image formats too like jpeg or tiff?
* wondering about having annotations on for a save at this size too? It'd be another md5 hash to check but as long as we confirm the image from which the baseline md5 hash is computed looks correct, then we'd know if annotations break at this scale
* Are there other *annotation*-like options worth testing here? Background color/gradient, ...
* Do we expect the md5 values to vary with platform and/or version of png/jpeg/tiff libs?

Sorry for posting now as I just see you merged it 😆

Reply:
Adding additional image formats would be easy.
The database, legend and axes are included in the image (no user or time information).
Other annotation-like options could be tested. Do you have any in mind?
The hashes would probably be different since the images would be different. It certainly isn't as robust as an image comparison with some type of fuzzy matching and I don't expect there would be any way to make the md5 sum matching fuzzy since I would expect a pixel difference to generate a significantly different md5 sum.

Reply:
> Do you have any in mind?

Not really... maybe background color/gradient...glad you are testing some here though so that is probably adequate.

I agree with your rationale....no reason to be a robust/fuzzy image-based compare for this test. We have plenty of other tests that *should* catch that. We just wanna test large images work and I was just thinking out loud whether there are any high level *mode*-like options that might play into that.